src/clojure/nrepl/logging.clj

```diff
@@ -0,0 +1,16 @@
+(ns nrepl.logging
+  "Logging to be used at nREPL"
+  {:added  "0.6.0"})
+
+;; TODO: Use clojure.tools.logging if available
```

Comment:
Seems to me the easiest way to do this would be to check if tools.logging is available, and defer in our to functions to its functions if that's the case. If not ours just dump stuff to stdout and stderr. I guess you're working in this direction already.